rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        sum(last_over_time(argocd_app_info{sync_status="Synced"}[5m]))
+        /
+        (
+          sum(last_over_time(argocd_app_info{sync_status="Synced"}[5m])) + sum(last_over_time(argocd_app_info{sync_status="OutOfSync"}[5m])) + sum(last_over_time(argocd_app_info{sync_status="Unknown"}[5m]))
```

Comment:
```suggestion
          sum(last_over_time(argocd_app_info{sync_status="Synced"}[5m])) + 
          sum(last_over_time(argocd_app_info{sync_status="OutOfSync"}[5m])) + 
          sum(last_over_time(argocd_app_info{sync_status="Unknown"}[5m]))
```

Reply:
I think this one doesn't necessarily sums up to the number of apps on the system. If an app changed state within the time frame, it will have last_over_time of 1 within the time frame for multiple statuses (as I don't think this metric ever returns something other than 1 - it either gives one or does not exist).

Reply:
I reverted it back to the original expression that I had, for now.  I replied to your comment.